src/docs/product/alerts/alert-types.mdx

```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
 
 ### Errors
 
-- Number of Errors
-- Users Experiencing Errors
+Error alerts are useful for monitoring the overall level of errors in your project, errors occurring in specific parts of your app, or errors affecting your users.
+
+#### Number of Errors
+
+Alerts when the number of errors in a project matching your filters crosses the threshold you set.
+
+#### Users Experiencing Errors
+
+Alerts when the number of users affected by errors in your project crosses the threshold you set.
 
 ### Sessions (Crash Rate Alerts)
 
-Crash rate alerts tell you when the crash free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash). Use crash rate alerts to monitor the health of your application per project, or in a specific release
+Crash rate alerts can give you a better picture of the health of your app on a per project basis or in a specific release. They trigger when the crash-free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash).
+
+#### Crash Free Session Rate
+
+Alerts when the number of crashed sessions exceeds the threshold you set.
+
+A session begins when a user starts the application and ends when it’s closed or sent to the background. A crash is when a session ends due to an error.
+
+#### Crash Free User Rate
 
-- Crash Free Session Rate
-- Crash Free User Rate
+Alerts when the overall user experience dips below the threshold you set.
+
+Crash Free Users represents the percentage of individual users who haven’t experienced a crash.
 
 ### Performance
 
-- Throughput
-- Transaction Duration
-- Apdex
-- Failure Rate
-- Largest Contentful Display
-- First Input Delay
-- Cumulative Layout Shift
-- Custom Metric
+Application performance alerts can help you pinpoint and identify specific problems that could be causing a suboptimal experience for your users.
+
+#### Throughput
+
+Alerts when throughput (the total number of transactions in a project) reaches a threshold set by you within a specified period of time.
+
+#### Transaction Duration
+
+This alert can be set to monitor how long it takes for transactions to complete. It can be customized with flexible aggregates like percentiles, averages, and min/max.
```

Comment:
Should we rephrase this to start with "Alerts on..." to match the others?

```suggestion
Alerts when transaction durations meet certain conditions. The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.
```

Reply:
Great point!